profiles/bicycle.lua

```diff
@@ -188,6 +188,15 @@ function setup()
     avoid = Set {
       'impassable',
       'construction'
+    },
+
+    node_tags_requiring_processing = {
+      'barrier',
+      'highway',
+      'crossing',
+      'bicycle',
+      'vehicle',
+      'access'
```

Comment:
a possibility would be to do test-run where you build a list of all tags looked at by the profile. 

or maybe it would be possible to keep all node-relevant tags in a single parent table (with sub tables as needed), so you can collect the total set of tags automatically? 

but in practice i think this is a small problem, since the process_node function is typically rather simple and is rarely modified.